include/PICA/pica_frag_config.hpp

```diff
@@ -81,6 +81,11 @@ namespace PICA {
 		LightingLUTConfig rg{};
 		LightingLUTConfig rb{};
 
+		u32 config1;
+		u32 lutAbs;
```

Comment:
These are duplicated, the LutConfig members should be changed to be an array and then we can access these attributes from there without duplicating them